src/Nest/Search/Search/Highlighting/Highlight.cs

```diff
@@ -115,5 +148,13 @@ public class HighlightDescriptor<T> : DescriptorBase<HighlightDescriptor<T> ,IHi
 		public HighlightDescriptor<T> BoundaryCharacters(string boundaryCharacters) => Assign(a => a.BoundaryChars = boundaryCharacters);
 
 		public HighlightDescriptor<T> BoundaryMaxSize(int boundaryMaxSize) => Assign(a => a.BoundaryMaxSize = boundaryMaxSize);
```

Comment:
https://github.com/elastic/elasticsearch/issues/1614 we're in good company of making this mistake :)